src/pages/vim/basic-usage/index.md

```diff
@@ -2,7 +2,8 @@
 title: Basic Usage
 ---
 
-## VIM Basic Usage
+<h1 align="center"> VIM Basic Usage </h1>
```

Comment:
Hi @sayantanHack 

Can you please revert this change back to how it was?

Markdown formatting should be used in all guide pages and not HTML formatting. Headings should different amounts of hashtags `##` to denote the heading type.